draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -1400,14 +1400,15 @@ deployed and not likely to be registered in a timely manner otherwise.</t>
    as quoted-string (<xref target="quoted.strings"/>).
 </t>
 <t>
-   Because commas (",") are used as a generic delimiter between field values,
-   they need to be treated with care if they are allowed in the field value.
-   Typically, components that might contain a comma are protected with
-   double-quotes using the quoted-string ABNF production.
+   Because commas (",") are used as a generic delimiter between members of a
+   list-based field value, they need to be treated with care if they are
+   allowed in those members. Typically, list members that might contain a
+   comma are protected with double-quotes using the quoted-string ABNF
```

Comment:
"are protected with double-quotes using the quoted-string ABNF production" -> "are enclosed in a quoted-string"